saleor/settings.py

```diff
@@ -58,24 +58,30 @@ def get_bool_from_env(name, default_value):
 TIME_ZONE = 'America/Chicago'
 LANGUAGE_CODE = 'en'
 LANGUAGES = [
+    ('ar', _('Arabic')),
     ('bg', _('Bulgarian')),
+    ('bn', _('Bengali')),
     ('cs', _('Czech')),
+    ('da', _('Danish')),
     ('de', _('German')),
     ('en', _('English')),
     ('es', _('Spanish')),
     ('fa', _('Persian')),
     ('fr', _('French')),
     ('hu', _('Hungarian')),
+    ('hy', _('Armenian')),
```

Comment:
It looks like Armenian is not valid language code in django

Reply:
Good catch!